tests/test_search.py

```diff
@@ -0,0 +1,81 @@
+import unittest
+from src import search
+import requests
+
+
```

Comment:
Rename blank to something else since you will incorporate a "void/blank" input test case.